src/SimpleExcelWriter.php

```diff
@@ -68,11 +68,11 @@ public static function streamDownload(
         $writer = $simpleExcelWriter->getWriter();
 
         if ($writerCallback) {
-            $writerCallback($writer);
+            $writerCallback($writer, $downloadName);
+        } else {
```

Comment:
Could you use a ternary instead of `else`?

Reply:
Yes, I've updated it.